examples/ex-offset.cpp

```diff
@@ -87,217 +91,228 @@
-      
-      box_ref(row,col) = box0(row,col) + box0(row-1,col)         
-                       + box0(row+1,col) + box0(row,col-1) + box0(row,col+1);
-      
-    }
-  }
-  
-  //printBox(box_ref,boxN);
-//----------------------------------------------------------------------------//
+  // printLattice(lattice_ref,latticeN);
+  //----------------------------------------------------------------------------//
 
-//
-//RAJA versions
-//
+  //
+  // RAJA versions
+  //
 
-//
-//Create loop bounds
-//
-  RAJA::RangeSegment col_range(0,N);
-  RAJA::RangeSegment row_range(0,N); 
-  
-//
-// Here we replace the macros with RAJA views and make utility of offset-layouts.
-//
+  //
+  // Create loop bounds
+  //
+  RAJA::RangeSegment col_range(0, N);
+  RAJA::RangeSegment row_range(0, N);
 
-//
-//Dimension of the array
-//
+  //
+  // Specify the dimension of the lattice
+  //
   const int DIM = 2;
 
-//
-//The first array corresponds to the coordinates of the bottom left corner
-//and the second array corresponds to the coordinates of the top right corner.
-//
-  RAJA::OffsetLayout<DIM> layout = RAJA::make_offset_layout<DIM>({{-1,-1}}, {{N,N}});
-  RAJA::View<int, RAJA::OffsetLayout<DIM>>box0view(box0,layout);
-  RAJA::View<int, RAJA::OffsetLayout<DIM>>box1view(box1,layout);
-  
-//----------------------------------------------------------------------------//
-  std::cout << "\n Running sequential box-stencil (RAJA-Kernel - sequential)...\n";  
-  using NESTED_EXEC_POL = 
-    RAJA::KernelPolicy<
-      RAJA::statement::For<1, RAJA::seq_exec,    // row
-        RAJA::statement::For<0, RAJA::seq_exec,  // col
-          RAJA::statement::Lambda<0>
-        >
-      >  
-    >;
-
-  RAJA::kernel<NESTED_EXEC_POL>
-    (RAJA::make_tuple(col_range, row_range), 
-     [=](int col, int row) {
-      
-      box1view(row,col) = box0view(row,col) + box0view(row-1,col) 
-                        + box0view(row+1,col) + box0view(row,col-1) + box0view(row,col+1);
-    });
-
-  //printBox(box1,boxN);
-  checkResult(box1, box_ref, boxSize);
+  // In the following snippet of code we introduce an offset layout and view
+  // to simplify multidimensional indexing.
+  // An offset layout is constructed by using the make_offset_layout method.
+  // The first argument of the layout is the coordinates of the bottom left
+  // corner,
+  // and the second array are the cordinates of the top right corner of our
+  // shifted
+  // lattice.
+  //
+  RAJA::OffsetLayout<DIM> layout = RAJA::make_offset_layout<DIM>({{-1, -1}}, {{N, N}});    
+  RAJA::View<int, RAJA::OffsetLayout<DIM>> lattice0view(lattice0, layout);
+  RAJA::View<int, RAJA::OffsetLayout<DIM>> lattice1view(lattice1, layout);
+
+  //----------------------------------------------------------------------------//
+  std::cout << "\n Running sequential five-box-stencil (RAJA-Kernel - "
+               "sequential)...\n";
+  using NESTED_EXEC_POL = RAJA::
+      KernelPolicy<RAJA::statement::
```

Comment:
Please format the nested policies (here and others below) using the indentation pattern and put each closing '>' on its own line as in the matrix multiplication examples in the ex4... file.  It makes the policies much easier to read and understand.